### PR TITLE
Bug/fix pause filter

### PR DESCRIPTION
**Pull request by Secretmapper**
Fix Pause showing all torrents

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "react-hot-loader/babel",
+    "babel-plugin-transform-decorators-legacy",
+    "babel-plugin-transform-object-rest-spread",
+    "babel-plugin-syntax-trailing-function-commas",
+    "babel-plugin-transform-export-extensions",
+    [
+      "module-resolver",
+      {
+        "root": ["./src"],
+        "alias": {
```
Comment (edulan) [inline .babelrc]: We can configure jest to resolve webpack alias, so there's no need to change babel configuration.

Example:
```json
"jest": {
  "moduleNameMapper": {
    "^util(.*)$": "<rootDir>/src/util$1",
    "^stores(.*)$": "<rootDir>/src/stores$1"
  }
}
```

More info https://facebook.github.io/jest/docs/webpack.html#configuring-jest-to-find-our-files

File: src/stores/__tests__/torrent-store.spec.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  STATUS_STOPPED,
+  STATUS_CHECK_WAIT,
+  STATUS_CHECK,
+  STATUS_DOWNLOAD_WAIT,
+  STATUS_DOWNLOAD,
+  STATUS_SEED_WAIT,
+  STATUS_SEED
+} = Torrent;
+
+const mapStatus = arr => lodash.map(arr, 'status')
```
Comment (edulan) [inline src/stores/__tests__/torrent-store.spec.js]: Using object destructuring:
```javascript
const mapStatus = arr => arr.map(({ status }) => status);
```

This way we avoid lodash dependency :wink:

**Comment by edulan** [comment]
Really appreciate your contribution @Secretmapper :clap: :clap:

File: src/stores/__tests__/torrent-store.spec.js (hunk 2 above)
Comment (Secretmapper) [reply src/stores/__tests__/torrent-store.spec.js]: I thought lodash was used throughout the project (didn't realize it only had one reference!) I defintely agree that we should keep it out :)

**Comment by edulan** [comment]
@Secretmapper can you please resolve conflicts?

**Comment by Secretmapper** [comment]
Fixed! Pretty silly error on my part!

File: package.json [hunk 3]
```diff
@@ -57,7 +57,6 @@
   },
   "dependencies": {
     "autobind-decorator": "1.3.4",
-    "lodash": "^4.17.2",
```
Comment (edulan) [inline package.json:60]: Why removing lodash?

Comment (edulan) [reply package.json:60]: Sorry :sweat_smile:, I meant that we shouldn't rely on lodash for things like `map`, as we can already perform that with just vanilla javascript.

Recent added features https://github.com/fcsonline/react-transmission/pull/29 rely on lodash for things like `zip`, so it'd be better to keep it to not conflict :wink:

File: src/stores/__tests__/torrent-store.spec.js [hunk 4]
```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import lodash from 'lodash';
```
Comment (edulan) [inline src/stores/__tests__/torrent-store.spec.js]: Not used :fire:

File: package.json (hunk 3 above)
Comment (Secretmapper) [reply package.json:60]: Ah if that's the case I agree that it'd be best to keep it.

**Pull request merged**